src/utils/api.js

```diff
@@ -97,6 +98,14 @@ export const getGames = async () => {
         published: true,
         ...perfGame.board,
       },
+      {
+        ...unpublishedGame,
+        name: "Unpublished Game",
+        data: unpublishedGame,
+        id: "unpublished",
+        published: false,
+        ...unpublishedGame.board,
```

Comment:
@jrmi why do you put this one at the end, instead of:
```
{
        ...unpublishedGame,
        ...unpublishedGame.board,
        name: "Unpublished Game",
        data: unpublishedGame,
        id: "unpublished",
        published: false,
}
```
That at least ensures that the values set here are not overwritten by the last `...unpublishedGame.board`. Setting `published: false` here is overwritten by the `published` field of the `...unpublishedGame.board`.

Reply:
Because I want to be able to overwrite this values with the board one. But I think it's just old compatibility code that may be removed now and  all data should be in the game file, don't you think?